reconcile/gitlab_housekeeping.py

```diff
@@ -306,7 +362,9 @@ def preprocess_merge_requests(
     dry_run: bool,
     gl: GitLabApi,
     project_merge_requests: list[ProjectMergeRequest],
+    state: State,
     users_allowed_to_label: Iterable[str] | None = None,
+    must_pass: Iterable[str] = [],
```

Comment:
don't use `[]` as default arg value as it's only eval once and will be shared in all function calls, use `None` or no default value at all